util/geosop/GeosOp.cpp

```diff
@@ -143,7 +143,8 @@ GeosOp::GeosOp(GeosOpArgs& arg)
 GeosOp::~GeosOp() {
 }
 
-std::string formatNum(long n)
+template<typename T>
+std::string formatNum(T n)
```

Comment:
Using an arbitrary type T is too general. What if T is `char`?

Reply:
> I was just going to make this unsigned long.

That works too, but then it would need to be called only with `unsigned long` arguments. Right now it is called with both signed and unsigned, so I went with the minimal change.

> What if T is char?

`char` tops out at 255, so it would get converted to a string and no commas would be added.

Reply:
> > I was just going to make this unsigned long.
> 
> That works too, but then it would need to be called only with `unsigned long` arguments. Right now it is called with both signed and unsigned, so I went with the minimal change.
The function only works with unsigned numbers. It should probably be called `formatCount` to reflect that.
> 
> > What if T is char?
> 
> `char` tops out at 255, so it would get converted to a string and no commas would be added.
Wouldn't to_string convert the char to a string?

Reply:
> The function only works with unsigned numbers. It should probably be called formatCount to reflect that.

Sounds good.

> Wouldn't to_string convert the char to a string?

No, see https://en.cppreference.com/w/cpp/string/basic_string/to_string